polars/polars-lazy/polars-plan/src/dsl/dt.rs

```diff
@@ -226,12 +226,14 @@ impl DateLikeNameSpace {
             .map_private(FunctionExpr::TemporalExpr(TemporalFunction::TimeStamp(tu)))
     }
 
-    pub fn truncate<S: AsRef<str>>(self, every: S, offset: S) -> Expr {
+    pub fn truncate<S: AsRef<str>>(self, every: S, offset: S, use_earliest: Option<bool>) -> Expr {
```

Comment:
Maybe we should make a `TruncateArgs` struct that implements `Default`. 

In any case, I don't think we should have an `Option<bool>`, but just a `bool`. The `Default` implementation can then decide what the `default` case should be.

Reply:
`Option<bool>` would mirror what's done for `replace_time_zone`, i.e.:
- `None`: just raise on ambiguous datetimes
- `true`: use the earliest datetime (e.g. `2020-01-01 01:00 BST`)
- `false`: use the latest datetime (e.g. `2020-01-01 01:00 GMT`)

I'll clarify this in the docs, and will try out making `TruncateArgs `